RNTester/js/TextExample.android.js

```diff
@@ -494,9 +494,13 @@ class TextExample extends React.Component<{}> {
             This text will have a orange highlight on selection.
           </Text>
         </RNTesterBlock>
-        <RNTesterBlock title="Inline images">
+        <RNTesterBlock title="Inline views">
```

Comment:
If inline views are now supported on both platforms we should consolidate the example out of this .android.js file so that the examples are used for both ios and android. 

Also, your PR summary lists some potential edge cases around clipping and different sized views. Can you add examples for not just the happy path but for all of those edge cases too? That way we can track how they change over time and ensure that they have consistent behavior across iOS and Android.

Reply:
Thanks for the suggestions.

I removed the example I added to RNTester for now because it won't work until we make a change to the React repo. Specifically, we have to remove [this assert from `ReactNativeRenderer-dev.js`](https://github.com/facebook/react-native/blob/536045276cd381aa37f72d9a5afa8ff68b40d575/Libraries/Renderer/oss/ReactNativeRenderer-dev.js#L3965-L3968).

I created a [branch with several inline view examples](https://github.com/rigdern/react-native/commit/ae11179a492283a7ab7bb7ef2b49e399eb8717c8) which I'll send as a PR after the prerequisite PRs have been merged. I shared code between iOS and Android where possible.

Unrelated to inline views, we should consider factoring out some of the common examples between `TextExample.ios.js` and `TextExample.android.js` into a shared file.